frontend/app/routes/protected/person-case/contact-information.tsx

```diff
@@ -208,42 +241,58 @@ export default function ContactInformation({ loaderData, actionData, params }: R
               options={countryOptions}
               errorMessage={errors?.country?.at(0)}
               defaultValue={loaderData.defaultFormValues?.country}
+              onChange={handleCountryChange}
```

Comment:
```suggestion
              onChange={({ target }) => setCountry(target.value)}
```

I don't see a reason to define another function just for this (makes the code harder to read).